akka-actor-typed-tests/src/test/scala/akka/actor/typed/internal/ActorSystemSpec.scala

```diff
@@ -49,9 +49,7 @@ class ActorSystemSpec extends WordSpec with Matchers with BeforeAndAfterAll
           }
           inbox.receiveAll() should ===("hello" :: Nil)
         }
-      val p = t.ref.path
-      p.name should ===("/")
-      p.address.system should ===(suite + "-a")
+      t should ===(Right(Done))
```

Comment:
Not sure this assertion matters anymore now, so maybe just have the withSystem return `Unit` instead.

Reply:
Thanks for review! As the purpose of the test case, if we eliminate the return value, how to ensure the sys termination is driven by guardian actor's termination?

Reply:
Hm, have it return the `Future[Done]` then and block on that here with await and a timeout?

Reply:
Sounds good!